ccmbioinfo/lfs
# LFS-290 As an administrator, I can set up an instance so that only specific users can access each record

To test the new owners permissions scheme, when **first** starting up a new instance, run with `-Dsling.run.modes=permissions_ownership`.

Things to test:
- create two other accounts
- create forms using one of the accounts, check that it can correctly see them in tables, filters, search
- create forms using one of the accounts, check that the other one cannot see them
- import data using one of the accounts, check that the other other one cannot see them, nor their associated subjects)

Given that it needs to be started with a custom parameter, I don't think this can be auto-deployed on a VM.

Also, startup failures are happening quite often, you may have to remove the sling folder and start it again a few times until it succeeds.
